src/novel_poly_basis.rs

```diff
@@ -12,100 +12,29 @@ use super::*;
 use super::f2e16::*;
 
 
-
+include!("params.rs");
 include!("encode.rs");
 include!("decode.rs");
```

Comment:
This should be avoided at all cost except for the table.

Reply:
No, they'll become separate modules eventually.  I'm just not ready to settle on exactly what the field functionality will be, but I'm kinda done working on them being in the same file.